staging/src/k8s.io/apiserver/pkg/audit/policy/reader.go

```diff
@@ -61,11 +62,22 @@ func LoadPolicyFromFile(filePath string) (*auditinternal.Policy, error) {
 
 func LoadPolicyFromBytes(policyDef []byte) (*auditinternal.Policy, error) {
 	policy := &auditinternal.Policy{}
+
+	strictDecoder := serializer.NewCodecFactory(audit.Scheme, serializer.EnableStrict).UniversalDecoder(apiGroupVersions...)
```

Comment:
Do you need to specify the `apiGroupVersions` here? I think omitting them just decodes to the internal version, which is what we're using anyway?
```suggestion
	strictDecoder := serializer.NewCodecFactory(audit.Scheme, serializer.EnableStrict).UniversalDecoder()
```